File: Java/Easy/N-th-Tribonacci-Number.java
class Solution {
    public int tribonacci(int n) {
        int a = 0;
        int b = 1;
        int c = 1;
        if (n == 0) {
            return a;
        }
        if (n < 2) {
            return b;
        }
        for (int i = 3; i<=n; i++) {
            int sum = a+b+c;
            a = b;
            b = c;
            c = sum;
        }
        return c;
        
    }
}

File: Python/Easy/N-th-Tribonacci-Number.py
class Solution:
    def tribonacci(self, n: int) -> int:
        a = 0
        b = 1
        c = 1
        if n == 0:
            return a
        if n < 2:
            return b
        for i in range(3,n+1):
            sumABC = a+b+c
            a = b
            b = c
            c = sumABC
        return c
